src/main/java/com/faforever/client/remote/FafServerAccessorImpl.java

```diff
@@ -458,10 +476,7 @@ private void parseServerObject(String jsonString) {
             .forEach(consumer -> consumer.accept(serverMessage));
         messageClass = messageClass.getSuperclass();
       }
-      for (Class<?> type : ClassUtils.getAllInterfacesForClassAsSet(messageClass)) {
-        messageListeners.getOrDefault(messageClass, Collections.emptyList())
-            .forEach(consumer -> consumer.accept(serverMessage));
-      }
```

Comment:
this loop made no sense at all

Reply:
it seems to be a left over

Reply:
look how the variable is not used in the loop and the same call is made in the loop above